examples/main.go

```diff
@@ -9,37 +9,67 @@ import (
 	"regexp"
 	"sort"
 	"strings"
+	"unicode"
+	"unicode/utf8"
 
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 )
 
 var nameVar string
+var nameTitleVar string
 var imageVar string
 
 var templates = []string{
 	"ci.yml", "docs_example.md", "example_test.go", "example.go", "go.mod", "go.sum", "Makefile", "tools.go",
 }
 
 func init() {
-	flag.StringVar(&nameVar, "name", "", "Name of the example, use camel-case when needed. Only alphabetical characters are allowed.")
+	flag.StringVar(&nameVar, "name", "", "Name of the example. Only alphabetical characters are allowed.")
+	flag.StringVar(&nameTitleVar, "title", "", "(Optional) Title of the example name, used to override the name in the case initials are present (Mongodb -> MongoDB). Use camel-case when needed. Only alphabetical characters are allowed.")
 	flag.StringVar(&imageVar, "image", "", "Fully-qualified name of the Docker image to be used by the example")
 }
 
 type Example struct {
 	Image     string // fully qualified name of the Docker image
 	Name      string
+	TitleName string // title of the name: i.e. "mongodb" -> "MongoDB"
```

Comment:
```suggestion
	TitleName string // title of the name: e.g. "mongodb" -> "MongoDB"
```

My mnemonic trick: e.g. means "example given" 😅

Reply:
My mnemo trick was: i.e. means "in example", but it's probably poor Spanglish 😅